#Jacob Smith 9/12/2019 ROS Summary Project Homework
# This is an action definition file, which has three parts: 
# the goal, the result, and the feedback.
#
# Part 1: the goal, to be sent by the client
#
#the amount to turn
int32 turnGoal
---
# Part 2: the result, to be sent by the server upon completion
#
# How much we turned
int32 turnResult
# How many updates we provided along the way
uint32 updatesSent
---
# Part 3: the feedback, to be sent periodically by the server during execution.
# The amount of time that has elapsed from the start
int32 turnSoFar
